include/public/osqp_api_functions.h

```diff
@@ -259,15 +259,20 @@ OSQP_API OSQPInt osqp_update_rho(OSQPSolver* solver,
 
 /* ------------------ Derivative stuff ----------------- */
 #ifdef OSQP_ENABLE_DERIVATIVES
-OSQP_API OSQPInt osqp_adjoint_derivative(OSQPSolver*    solver,
-                                         OSQPFloat*     dx,
-                                         OSQPFloat*     dy_l,
-                                         OSQPFloat*     dy_u,
-                                         OSQPCscMatrix* dP,
-                                         OSQPFloat*     dq,
-                                         OSQPCscMatrix* dA,
-                                         OSQPFloat*     dl,
-                                         OSQPFloat*     du);
+OSQP_API OSQPInt osqp_adjoint_derivative_compute(OSQPSolver*    solver,
```

Comment:
It would be great if you could add the Doxygen-style comments to these three functions as well to make clear the inputs/outputs of the functions and what each argument does.

Reply:
Comments added